_includes/assets/sass/components/_internal-pages.scss

```diff
@@ -0,0 +1,19 @@
+.internal-page {
+    @include tbds-small {
+        width: 100%;
+        max-width: $tbds-breakpoint-small;
+        margin: auto;
+    }
+    h1 {
+        text-align: center;
+        font-size: 3rem;
+    }
+    h2 {
+        font-size: 1.75rem;
+        line-height: 2.09rem;
+    }
+    .introduction {
+        text-align: center;
```

Comment:
I think we could remove the center alignment to be more consistent with the homepage's left-aligned elements. What do you think?

Reply:
For context: https://github.com/thoughtbot/design-sprint-guide/pull/72#issuecomment-1353457748